Formula/elvis.rb

```diff
@@ -0,0 +1,44 @@
+class Elvis < Formula
+  desc "Erlang Style Reviewer"
+  homepage "https://github.com/inaka/elvis"
+  url "https://github.com/inaka/elvis/archive/refs/tags/1.1.0.tar.gz"
+  sha256 "4afa3629f95997449ec9ce15a4aa59dd8fece5a5320e23e1d1c7590d1831d953"
+  license "Apache-2.0"
+  head "https://github.com/inaka/elvis.git", branch: "master"
+
+  livecheck do
+    url :stable
+    strategy :github_latest
+  end
+
+  depends_on "rebar3" => :build
+  depends_on "erlang"
+
+  def install
+    system "printenv"
```

Comment:
Is this just printing the environment?

Reply:
Sorry, debug leftover. When I tried `brew install -si` the entire PATH got removed, so I wanted to see what it was supposed to be (with the super env stuff).